### PR TITLE
Replace model diagrams

### PR DESCRIPTION
**Pull request by hflynn**
See https://github.com/openmicroscopy/design/issues/54 and the Data Model detail column on https://trello.com/b/fMBThxpK/docs-comms

Staged at https://www.openmicroscopy.org/site/support/ome-model-staging/developers/model-overview.html and following pages.

Remaining questions:
- on https://www.openmicroscopy.org/site/support/ome-model-staging/developers/screen-plate-well.html are the cross-links diagrams helpful or do we want to add a line or two of text to https://www.openmicroscopy.org/site/support/ome-model-staging/developers/screen-plate-well.html#spw-schema-structure and then delete the rest of the page?
- on https://www.openmicroscopy.org/site/support/ome-model-staging/developers/roi.html is the additional shape diagram helpful or should I remove it as all the info is in the text? Gus designed a replacement diagram for the info written out in https://www.openmicroscopy.org/site/support/ome-model-staging/developers/roi.html#shape-attributes-and-simple-children too but I've assumed that really is adding nothing given it's just an abbreviated version of the text
- do we want to bother updating the diagrams on https://www.openmicroscopy.org/site/support/ome-model-staging/developers/legacy/tiled-images.html given they are a legacy solution? Gus started working on something on the design issue but it seemed like Josh wasn't entirely satisfied with it

(N.B. when we make a decision re: cross-link diagrams, I'll delete the existing ones and also any remaining pdf images on the assumption we are ditching the pdf for these docs when they move to their new home)

We also need to decide where to keep the omnigraffle file Gus used to generate these - in the repo or on squig somewhere?

cc/ @gusferguson @mtbc @joshmoore

### CODE REVIEW

**Comment by hflynn** [comment]
Actually is https://github.com/openmicroscopy/design/issues/54#issuecomment-247936630 intended to replace the crosslink diagrams? Does it do a sufficient job?

File: formats/developers/structured-annotations.txt [hunk 1]
```diff
@@ -22,14 +22,14 @@ Key-value pairs <developers/Model/KeyValuePairs.html>` and a
...
-.. figure:: /images/Annotation-Top.*
+.. figure:: /images/structured_annotation_branch.png
    :align: center
    :alt: StructuredAnnotation Model branch
 
    The StructuredAnnotation branch of the OME Model
 
 
-.. figure:: /images/Annotation-Points.*
+.. figure:: /images/annotation_points.png
```
Comment (dgault) [inline formats/developers/structured-annotations.txt:32]: I may wrong but I couldn't find an AnnotationRef on Experiment, it appears like it has always been in the diagrams though

**Comment by dgault** [comment]
The updated images all look really good and are loading and displaying correctly.

I actually think the new Shape diagram is really useful and my preference would be to keep it alongside the text even if they do describe the same thing.

As for the cross linked diagrams, personally I would find it helpful having the diagrams but other people might have different opinions.

**Comment by hflynn** [comment]
Does https://github.com/openmicroscopy/design/issues/54#issuecomment-247936630 help you at all with envisioning the links for spw @dgault ?

File: formats/developers/structured-annotations.txt (hunk 1 above)
Comment (hflynn) [reply formats/developers/structured-annotations.txt:32]: @mtbc perhaps you'll look at this when you're back, I'll need to get @gusferguson to revise the diagram if you agree.

**Comment by dgault** [comment]
Im perhaps looking at these from more of a developer perspective. The new diagram from Gus (https://github.com/openmicroscopy/design/issues/54#issuecomment-247936630) is very good at helping visualise what exactly the data is representing, whereas the first of the cross link diagrams (https://www.openmicroscopy.org/site/support/ome-model-staging/_images/Spw-structure-cross-linked.png) is also pretty useful from a developers perspective to show the references within the model structure.

Perhaps the combination of these two diagrams would be enough without needing the last 2 sections (Sample SPW Schema file structure and Cross-linked sample file structure).

**Comment by gusferguson** [comment]
"the cross link diagrams (https://www.openmicroscopy.org/site/support/ome-model-staging/_images/Spw-structure-cross-linked.png)" 

has been updated and is shown in the page:

https://www.openmicroscopy.org/site/support/ome-model-staging/_images/hcs_structures.png

**Comment by dgault** [comment]
Yep, sorry posted the wrong link - https://www.openmicroscopy.org/site/support/ome-model-staging/_images/hcs_structures.png along with https://github.com/openmicroscopy/design/issues/54#issuecomment-247936630 to me provides a good combination of the information

**Comment by hflynn** [comment]
Excellent, I'll push another commit and add that extra diagram, remove the old cross-links ones and amend the text

**Comment by hflynn** [comment]
https://www.openmicroscopy.org/site/support/ome-model-staging/developers/screen-plate-well.html updated @dgault - better?

Assuming so, remaining issues are whether you can annotate experiment and where to keep the graffle file

**Comment by dgault** [comment]
Yeah I really like the look of the new SPW page, seems like a good improvement to me.

File: formats/developers/structured-annotations.txt (hunk 1 above)
Comment (mtbc) [reply formats/developers/structured-annotations.txt:32]: Confirmed, there's no `AnnotationRef` on `Experiment`. (Gah, GH links to these comments never seem to work!)

Comment (hflynn) [reply formats/developers/structured-annotations.txt:32]: @gusferguson can you update this one diagram for me please?

Comment (gusferguson) [reply formats/developers/structured-annotations.txt:32]: @hflynn - is this what you need?

annotation_points

Comment (hflynn) [reply formats/developers/structured-annotations.txt:32]: Yes, that's great thanks @gusferguson

**Comment by hflynn** [comment]
Rebuilding staging now but this should be good to merge now, if you want to sign off @joshmoore 

Looking at the size of the graffle file, I've put it in team>Documentation>Specification>Diagrams for now

**Comment by joshmoore** [comment]
Makes sense to me. Thanks, all -- very nice.

**Pull request merged**